### DISCUSSION TITLE
Do I need a VPS to deploy a website that uses Pocketbase?

### DISCUSSION THREAD

**Discussion by getgizmo** [pocketbase/pocketbase#1565] [Q&A]
Imagine I want to make my website live, which uses Pocketbase. Is there any service that helps with this or do I just get a VPS and put everything there?

**Comment by ganigeorgiev** [comment]
You need a VPS with a persistent storage, where you can run the PocketBase executable and open port(s).

For a generic guide, you can check - https://pocketbase.io/docs/going-to-production/#minimal-setup.

> Is there any service that helps with this?

You can have a look at [pockethost.io](https://pockethost.io/), but I haven't really tested it so I cannot provide help with it. You can find their repo in https://github.com/benallfree/pockethost.
